pictures/models.py

```diff
@@ -102,15 +102,23 @@ def update_all(self, from_aspect_ratios):
     @property
     def width(self):
         self._require_file()
-        if self._committed and self.field.width_field:
+        if (
+            self._committed
+            and self.field.width_field
+            and hasattr(self.instance, self.field.width_field)
```

Comment:
Based on those changes, I assume the width and height field are cleaned as well? Maybe you want to include an assertion for that in the tests?